tests/drivers/build_all/sensor/spi.dtsi

```diff
@@ -337,3 +337,32 @@ test_spi_icm426888: icm42688@28 {
 	gyro-hz = <32000>;
 	gyro-fs = <2000>;
 };
+
+#include <zephyr/dt-bindings/adc/adc.h>
+
+test_spi_ads7052: ads7052@29 {
+	compatible = "ti,ads7052";
+	reg = <0x29>;
+	spi-max-frequency = <12000000>;
+	#io-channel-cells = <1>;
+	#address-cells = <1>;
+	#size-cells = <0>;
+	channel@0 {
+		reg = <0>;
+		zephyr,gain = "ADC_GAIN_1";
+		zephyr,reference = "ADC_REF_VDD_1";
+		zephyr,vref-mv = <3300>;
+		zephyr,acquisition-time = <ADC_ACQ_TIME_DEFAULT>;
+		zephyr,resolution = <14>;
+	};
+};
+
+test_spi_b57861s0103a039: b57861s0103a039@2a {
```

Comment:
This should probably go into a new adc.dtsi which is included in app.overlay, @MaureenHelm thoughts?